src/main/java/seedu/todo/controllers/concerns/Renderer.java

```diff
@@ -14,6 +14,22 @@
 public class Renderer {
     
     /**
+     * Renders an error message in both the console and the input field, leave null or empty string if not needed.
+     * 
+     * @param consoleMessage       Message to be rendered in the console
+     * @param consoleInputValue    Value to display in the input field
+     */
+    public static void showConsoleError(String consoleMessage, String consoleInputValue) {
```

Comment:
Hmm, can we abstract this further? Every controller will need to implement a disambiguation method, and I don't think MESSAGE_DISAMBIGUATION will be different for each controller. Or is this a TODO for me to write more detailed console errors saying exactly what went wrong?